modules/runtime/nuxeo-launcher-commons/src/main/java/org/nuxeo/launcher/config/ConfigurationGenerator.java

```diff
@@ -643,25 +605,6 @@ protected void includeTemplates() throws IOException {
         log.debug(includedTemplates);
     }
 
-    private void logDebugInformation() {
-        String devPropValue = userConfig.getProperty(NUXEO_DEV_SYSTEM_PROP);
-        if (Boolean.parseBoolean(devPropValue)) {
-            log.debug("Nuxeo Dev mode enabled");
-        } else {
-            log.debug("Nuxeo Dev mode is not enabled");
-        }
```

Comment:
Not really, they appear when the launcher has the option `--debug`, I didn't find them really useful. Wdyt?

Reply:
maybe to be moved to trace; I remember that the dev mode introduced many code branches, maybe making it worth to warn the developer about the dev mode status?

The debug launcher option is a distinct subject, very much useful and missing logs since the move to Log4j2.

I would keep those logs unless the dev mode was removed. That's the purpose of a debug option to log everything relevant.

Reply:
Moving them to trace will definitively hide them.

We're warning the developer when the dev mode is enabled in the docker packaging, as it is the target I would say it is good to leverage the docker warn.

If it's not enough, then we can keep the _Nuxeo Dev mode is enabled_ log but in this case, I would log it at info level.

Wdyt?

Reply:
I kept these some (long) time ago when cleaning up the different "dev mode" notions (with "test", "debug", and "strict" modes...).
When in dev mode, some of the code logic is not optimized so i think it makes sense to keep one log when it's enabled, and remove the other one, as you're suggesting :+1:.
I'd use the warn level instead of info (and it would be consistent with the docker packaging behaviour)